File: src/main/java/Commands/Action.java
package ru.javarush.cryptoanalyzer.konovalov.Commands;

import ru.javarush.cryptoanalyzer.konovalov.entity.Result;

public interface Action {
    String ENCODE = "encode";
    String DECODE = "decode";
    String BRUTEFORCE = "bruteforce";
    String ANALYZE = "analyze";
    String EXIT = "exit";

    Result execute(String[] parameters);
}

File: src/main/kotlin/commands/Action.kt
package commands

import entity.Result

interface Action {

    companion object{
        val ENCODE: String = "encode"
        val DECODE: String = "decode"
        val BRUTEFORCE: String = "bruteforce"
        val ANALYZE: String = "analyze"
        val EXIT: String = "exit"
    }

    fun execute(parameters: Array<String>): Result

}